.circleci/config.yml

```diff
@@ -279,8 +279,15 @@ commands:
                 command: |
                   COMMIT_SHA=$(git rev-parse --short HEAD)
                   LAST_RELEASE=$(git describe --tags --abbrev=0)
+                  if [[ $LAST_RELEASE == android-v* ]]; then
+                    LAST_RELEASE=${LAST_RELEASE:9}
+                  elif [[ $LAST_RELEASE == extension-androidauto-v* ]]; then
+                    LAST_RELEASE=${LAST_RELEASE:23}
```

Comment:
This one will give `0.2.0` since auto extension follows separate versioning for now. We need to filter tags and lookup the last v* or android-v*

Reply:
Checked  and it seems multiple `--match` expressions are supported, so this should work : 
`git describe --tags --abbrev=0 --match 'android-v[0-9]*.[0-9]*.[0-9]*' --match 'v[0-9]*.[0-9]*.[0-9]*'` will return the last one that matches

Reply:
actually, i excluded `extension-androidauto` for now.  I guess if this match works, i can use it as well. 😉